audit_log/apps.py

```diff
@@ -0,0 +1,8 @@
+from django.apps import AppConfig
+from django.utils.translation import gettext_lazy as _
+
+
+class AuditLogConfig(AppConfig):
+    default_auto_field = "django.db.models.BigAutoField"
```

Comment:
Any reason to use int instead of uuid for the mdoel IDs?

Reply:
This line is actually auto-generated by the `startapp` management command. It's related to [this change](https://docs.djangoproject.com/en/3.2/releases/3.2/#customizing-type-of-auto-created-primary-keys) in Django 3.2.

It just tells Django which type of ID field to use as default if the model itself doesn't define a primary-key field (which is quite usual). As far as I know adding UUIDField here would not work, it would most likely require some `UUIDAutoField` implementation.

Reply:
Ahhh right, then maybe the wrong place to ask, but the question was about the `AuditLog` model.

Reply:
No particular reason really, I just went with the default. Would we like to have UUID as a primary key on the log entries? :thinking:

Reply:
Well I guess it doesn't really make a difference. I usually prefer using UUIDs as PKs but that's more a matter of personal preference 😅